docs/api/guides/guide-to-creating-an-endpoint.md

```diff
@@ -273,6 +273,28 @@ If you also need the session, you can do all three at once:
 appCtx := appcontext.WithSession(appcontext.NewAppContext(h.DB(), logger), session)
 ```
 
+### Validating Response Payload
+
+We need to ensure that the data that is being returned from the
+handler matches what is defined in the swagger definition.
+
+In your tests you should call `Validate` on the payload.  Something
+like ...
+
+
+```
```

Comment:
Let's add some line-highlighting to call out where the `.Validate` method is called and also some language syntax highlighting as well.

```suggestion
```go {8,9}
```